js/ui/windowManager.js

```diff
@@ -1913,7 +1912,6 @@ var WindowManager = new Lang.Class({
             Tweener.removeTweens(actor);
             actor.set_scale(1.0, 1.0);
             actor.set_opacity(255);
-            actor.set_pivot_point(0, 0);
```

Comment:
A quick heads up that resetting the pivot point was required for some of the affine transformation animations in https://github.com/endlessm/gnome-shell/pull/286 . I know that's not merged in now, so it shouldn't *break* anything, but still wanted to flag it here in case that work gets revisited and happens to be broken :)